reviews/methods1.Rmd

```diff
@@ -0,0 +1,450 @@
+
+> The point of not using a Type 1 error rate is to differentiate the statistical test from the the researcher's scientific judgement.
+> Type 1 error rate is a property of a test under certain conditions.
+> Theoretical risk refers to the individual judgement of a researcher about the substantive hypothesis and the study design as a whole.
+
+The authors say they borrowed the term 'theoretical risk' from Meehl, but that he did not assign it to (¬\|¬).
+The authors cite Kukla (1990), but after reading it, I am not convinced they are correct in equating 'theoretical risk' with the Type 1 error rate.
+Theoretical risk means making an a-priori unlikely prediction if the theory was not true.
+
+> This sentence sounds exactly like P(¬E|¬H) to us.
+> The judgement about P(¬E|¬H) is a-priori, is about "prediction" as a statement about evidence E, given that the theory is not true (¬H).
+> One might argue, that it should be multiplied by ¬H perhaps? 
+> But that would not change any of the arguments.
+> In any case, we provide clear mathematical definitions for our terms.
+> If a reader does not agree with our terminology, they are free to term them differently.
+> In fact, this happens in reference to a finding we found in the meantime (Oberauer and Lewandowsky (2019)), which we added to the discussion.
+
+This could be, as Meehl would like, a range prediction. Or, in the example of the authors on page 13, it is a test without confounds.
+In Lakens, 2019, on page 224, I explain how a two-sided test, one-sided test, and a range prediction all differ in how severe they test the hypothesis, even if they all have the same 5% alpha level ("Note that the three tests differ in severity even when they are tested with the same Type 1 error rate.").
+
+> Yes, indeed, this could all have low theoretical risk.
+> We could include the disclaimer to the examples on p. 13 "Note that the three tests differ in theoretical risk even when they are tested with the same Type 1 error rate."
+
+This is another reason why I think the term 'theoretical risk' is not a good choice.
+Indeed, based on the example on page 13, it seems the authors themselves confuse real theoretical risk with Type 1 error control.
+However, it is better to use terminology that encompasses Type 1 and Type 2 errors (such as 'the probability that a test will falsify a hypothesis', or even better: the *severity* of a test).
+
+> Different philosophies differ in their terminology.
+> Deborah Mayo has good reasons to differentiate Type 1 error rate from severity, as do we.
+
+Using the term 'severity of the test' would also embed the work more strongly in philosophy of science (and specifically the work of Mayo, as she builds on Popper).
+
+> This would embed us in a different philosophy of science.
+> Popper is the most famous anti-inductionist; we deliver an inductionist account.
```

Comment:
I believe that really here the core difference in lines of reasonings becomes apparent. It's really all about adressing this very fundamental issue. I think that this point should be adressed very early on both in the responses to the reviewer as well as in the manuscript.